api/envoy/extensions/filters/http/local_ratelimit/v3/local_rate_limit.proto

```diff
@@ -91,4 +91,6 @@ message LocalRateLimit {
   //
   //  The filter supports a range of 0 - 10 inclusively for stage numbers.
   uint32 stage = 9 [(validate.rules).uint32 = {lte: 10}];
+
+  bool per_connection = 10;
```

Comment:
Let's put a long description her explaining what this knob does, when it might be a good idea to use it, etc, etc.

Reply:
Would it make sense to make this name a little more descriptive, such as `rate_limiter_per_connection` maybe?

Reply:
Yes, I think that would help. I think `rate_limiter_per_connection` is good, given that we have `connection_pool_per_downstream_connection` in the cluster msg definition. Alternatively `apply_per_connection` is a suggestion if you are unconvinced.

Reply:
Alternatively, `per_connection_local_rate_limiter` is also good. I think I like that one the most.